workflow/cases/prototype_5.1.yaml

```diff
@@ -0,0 +1,60 @@
+case:
```

Comment:
This isn't needed at this time.  We are not running a prototype 5.1 yet.  The next one will be p6.  We'll make a case file for it when we are ready, but we are not there yet.